eventtracking/backends/segment.py

```diff
@@ -69,6 +74,13 @@ def send(self, event):
         path = context.get('path')
         referer = context.get('referer')
         page = context.get('page')
+
+        if path and not page:
+            # Try to put together a url from host and path:
+            host = context.get('host')
+            if host:
+                page = urljoin("//{host}".format(host=host), path)
+
```

Comment:
Is there already test coverage for this edge case somewhere?

Reply:
This is why we have code reviews.   Added a test.